src/main/java/Duke.java

```diff
@@ -1,10 +1,138 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    public static void printLine(int n) {
+        System.out.println("\t" + "-".repeat(n));
+    }
+
+    public static void printGreeting() {
+        printLine(60);
+        System.out.println("\t" + "Hello! I'm Duke \n    What can I do for you?");
+        printLine(60);
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        printGreeting();
+        Task[] tsk = new Task[100];
+        Scanner in = new Scanner(System.in);
+        String input;
+        input = in.nextLine();
+        String command;
+//        System.out.println(Task.getTaskCount());
+        int spacePos;
+        String eventDone;
+
+        while (!input.equals("bye")) {
+            spacePos = input.indexOf(" ");
+            command = spacePos > 0 ? input.substring(0, spacePos) : input;
+            printLine(60);
+            switch (command) {
+            case "done":
+                eventDone = input.substring(spacePos + 1);
+                taskDone(tsk, eventDone);
+                break;
+            case "list":
+                //print list of items
+                listTasks(tsk);
+                break;
+            case "todo":
+                String toDoTask = input.substring(spacePos + 1);
+                addToDo(tsk, toDoTask);
+                break;
+            case "deadline":
+                String deadlineTask = input.substring(spacePos + 1);
+                addDeadline(tsk, deadlineTask);
+                break;
+            case "event":
+                String eventTask = input.substring(spacePos + 1);
+                addEvent(tsk, eventTask);
+                break;
+            default:
+                break;
+            }
+            printLine(60);
+            input = in.nextLine();
+        }
+        exit();
+    }
+
+    public static void exit() {
+        printLine(60);
+        System.out.println("\t" + "Bye! Hope to see you again soon!");
+        System.out.println("\tYou currently have " + Task.getTaskLeft() + " tasks left");
+        System.out.println("\t" + "Happy to help you organize work. Anywhere, anytime!");
+        printLine(60);
+    }
+
+    private static void addEvent(Task[] tsk, String eventTask) {
+        int slot = eventTask.indexOf("/");
+        String description = eventTask.substring(0, slot - 1);
+        Event newItem = new Event(description, eventTask.substring(slot + 4));
+        tsk[Task.getTaskCount() - 1] = newItem;
+
```

Comment:
The blank line may not be necessary, but noted that you did this once in another add function. Also noted other functions with print lines don't have the blank line between the print section and the rest of the function. Maybe can standardise no blank line?